src/app/tests/TestCommandInteraction.cpp

```diff
@@ -1106,15 +1106,11 @@ void TestCommandInteraction::TestCommandHandlerInvalidMessageAsync(nlTestSuite *
     err = commandSender.SendCommandRequest(ctx.GetSessionBobToAlice());
     NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
 
-    // Warning! This test is not testing what the testname is suggesting. Because the invoke request is malformed,
-    // DispatchSingleClusterCommand is never called and asyncCommandHandle is never set. Possible fix to this test
-    // might be incoming when CommandSender is capable of sending multiple invoke command requests.
```

Comment:
But ... this does not fix the test, does it?  Why is this comment being removed?

Reply:
The point is: is the test now testing what the test name says it tests?  If not, there should still be a comment pointing out that the test is bunk and not testing what it's supposed to.